x-pack/plugins/security_solution/public/common/lib/triggers_actions_ui/register_alerts_table_configuration.tsx

```diff
@@ -29,15 +33,66 @@ const registerAlertsTableConfiguration = (
   const columnsFormStorage = dataTableStorage?.[TableId.alertsOnAlertsPage]?.columns ?? [];
   const alertColumns = columnsFormStorage.length ? columnsFormStorage : getColumns();
 
+  const useInternalFlyout = () => {
+    const { header, body, footer } = useToGetInternalFlyout();
+    return { header, body, footer };
+  };
+
+  const renderCellValueHookAlertPage = getRenderCellValueHook({
+    scopeId: SourcererScopeName.detections,
+  });
+
+  const renderCellValueHookCasePage = getRenderCellValueHook({
+    scopeId: SourcererScopeName.detections,
+  });
+
+  const sort: AlertsTableConfigurationRegistry['sort'] = [
+    {
+      '@timestamp': {
+        order: 'desc',
+      },
+    },
+  ];
+
+  // register Alert Table on Alert Page
   registry.register({
-    id: APP_ID,
+    id: `${APP_ID}`,
```

Comment:
```suggestion
    id: `${APP_ID}-alerts-page`,
```
?

Reply:
Sure.. to make it more clear. Will do this and corresponding changes.

Reply:
It is done now.